src/System.Drawing.Primitives/src/System/Drawing/Size.cs

```diff
@@ -74,6 +74,62 @@ public Size(int width, int height)
         public static Size operator -(Size sz1, Size sz2) => Subtract(sz1, sz2);
 
         /// <summary>
+        /// Multiplication operator that multiplies <see cref="Size"/> by an <see cref="int"/> producing <see cref="Size"/>.
+        /// </summary>
+        /// <param name="left">Multiplier of type <see cref="int"/>.</param>
+        /// <param name="right">Multiplicand of type <see cref="Size"/>.</param>
+        /// <returns>Product of type <see cref="Size"/>.</returns>
+        public static Size operator *(int left, Size right) => Multiply(right, left);
+
+        /// <summary>
+        /// Multiplication operator that multiplies <see cref="Size"/> by an <see cref="int"/> producing <see cref="Size"/>.
+        /// </summary>
+        /// <param name="left">Multiplicand of type <see cref="Size"/>.</param>
+        /// <param name="right">Multiplier of type <see cref="int"/>.</param>
+        /// <returns>Product of type <see cref="Size"/>.</returns>
+        public static Size operator *(Size left, int right) => Multiply(left, right);
+
+        /// <summary>
+        /// Division operator that divides <see cref="Size"/> by an <see cref="int"/> producing <see cref="Size"/>.
+        /// </summary>
+        /// <param name="left">Dividend of type <see cref="Size"/>.</param>
+        /// <param name="right">Divisor of type <see cref="int"/>.</param>
+        /// <returns>Result of type <see cref="Size"/>.</returns>
+        public static Size operator /(Size left, int right)
+        {
+            int invDiv = 1 / right;
+            return new Size(unchecked(left._width * invDiv), unchecked(left._height * invDiv));
```

Comment:
Actually, this one we shouldn't do multiplication by reciprocal (my fault for tagging this in the original pass). Integers can't do this optimization because 1 / x where |x| > 0 always results in 0. Multiplication by reciprocal is only a valid optimization when dealing with floating-point numbers.